test/projection.jl

```diff
@@ -212,7 +212,7 @@ struct NoSuperType end
         @test ProjectTo(I)(123) === NoTangent()
         @test ProjectTo(2 * I)(I * 3im) === 0.0 * I
         @test ProjectTo((4 + 5im) * I)(Tangent{typeof(im * I)}(; λ = 6)) === (6.0 + 0.0im) * I
-        @test ProjectTo(7 * I)(Tangent{typeof(2I)}()) == ZeroTangent()
+        @test ProjectTo(7 * I)(Tangent{typeof(2I)}()) == 0.0I
```

Comment:
hmm not sure that this is a good thing.
Does this also happen to other matrix types?